mesheryctl/internal/cli/root/system/system_test.go

```diff
@@ -75,6 +75,22 @@ func TestSystemCmdIntegration(t *testing.T) {
 			ExpectError:     false,
 			TimeoutRequired: 0,
 		},
+		// start and skip opening of meshery in browser
+		{
+			Name:            "Start Meshery with Docker platform with skip flag",
+			Action:          "start",
+			Args:            []string{"start", "-p", "docker", "-y", "-s"},
+			ExpectError:     false,
+			TimeoutRequired: 1,
+		},
+		//stop
+		{
+			Name:            "Stop Meshery with Docker platform",
+			Action:          "stop",
+			Args:            []string{"stop", "-y"},
+			ExpectError:     false,
+			TimeoutRequired: 0,
+		},
```

Comment:
Can you reset this file with master? There's isn't a need to add a specific test case for such a small flag, integration tests are really heavy and time-taking so it doesn't make sense to run them again with a flag.

Reply:
Removed test for flag